### PR TITLE
Add table RequestOfMe to UserRequestPage

### PR DESCRIPTION
**Pull request by ntdat213**

### CODE REVIEW

File: src/app/pages/request-user/request-user.component.html [hunk 1]
```diff
@@ -1,91 +1,92 @@
...
+    <div nz-col nzSpan="8">
+        <img style="width: 86%;
+      height: auto;" src="https://3.bp.blogspot.com/-5p7HmChO9vM/WWTVsHYyvSI/AAAAAAAAANQ/fdr_J7v6ZSkTzYYtT9KI74BX4_mk2LrgQCLcBGAs/s1600/19744842-The-word-Requests-on-a-suggestion-box-for-collecting-ideas-on-your-wants-desires-and-needs-Stock-Photo.jpg"
+        />
+    </div>
 </div>
+
+<div>
+    <h1>
+        REQUESTS OF ME
```
Comment (thanglenova) [inline src/app/pages/request-user/request-user.component.html]: Please rename to `My request`

File: src/app/pages/request-user/service/request-user.service.ts [hunk 2]
```diff
@@ -19,4 +19,9 @@ export class RequestUserService {
     const api = `${environment.apiUrl}/requests`;
     return this.http.post(api, request);
   }
+
+  getRequestOfMe():Observable<Request>{
+    const api = `${environment.apiUrl}/requests/getRequestOfMe`;
```
Comment (thanglenova) [inline src/app/pages/request-user/service/request-user.service.ts]: please rename both backend & frontend `/getRequestOfMe` to `/me`

File: src/app/pages/request-user/request-user.component.html (hunk 1 above)
Comment (ntdat213) [reply src/app/pages/request-user/request-user.component.html]: Done

File: src/app/pages/request-user/service/request-user.service.ts (hunk 2 above)
Comment (ntdat213) [reply src/app/pages/request-user/service/request-user.service.ts]: Done

**Pull request merged**